src/main/webapp/app/shared/components/code-button/code-button.component.ts

```
@@ -347,4 +356,75 @@
             this.user.vcsAccessToken = this.activeParticipation.vcsAccessToken;
         }
     }
+
+    initTheia(profileInfo: ProfileInfo) {
+        // The online IDE is only available with correct SpringProfile and if it's enabled for this exercise
+        if (profileInfo.activeProfiles?.includes(PROFILE_THEIA) && this.exercise) {
+            // Theia requires the Build Config of the programming exercise to be set
+            this.programmingExerciseService.getBuildConfig(this.exercise.id!).subscribe((buildConfig) => {
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)